api/types/types.go

```diff
@@ -240,26 +240,26 @@ type ContainerJSONBase struct {
 	ProcessLabel    string
 	AppArmorProfile string
 	ExecIDs         []string
-	HostConfig      *runconfig.HostConfig
+	HostConfig      runconfig.HostConfig
 	GraphDriver     GraphDriverData
 }
 
 type ContainerJSON struct {
-	*ContainerJSONBase
+	ContainerJSONBase
 	Mounts []MountPoint
-	Config *runconfig.Config
```

Comment:
Config is *VERY* big structure with maps and slices. This should be pointer according to Google style guide https://github.com/golang/go/wiki/CodeReviewComments